### ISSUE TITLE
Are we still using MacPython/h5py-wheels?

### ISSUE THREAD

**Issue by aragilar** [h5py/h5py#2026]
@tacaswell @takluyver Not sure if you're subscribed to the numpy-discussion mailing list, but apparently the MacPython organisation has run out of Travis credits: https://mail.python.org/archives/list/numpy-discussion@python.org/message/UTAVDGFTNOTDRPRS34ESE4PASOVVZNX5/

I'm guessing for the x86 platforms we're using azure-pipelines for wheels now, so we can drop those builds, but do we use MacPython for other systems?

**Comment by mattip** [comment]
For reference, the relevant link to the travis builds is https://app.travis-ci.com/github/MacPython/h5py-wheels/builds and is running a weekly cron job. I am not sure who has access to https://app.travis-ci.com/github/MacPython/h5py-wheels/settings where the cron job setting lives. I have access if you need help configuring it.

The build is driven by travis.yml at https://github.com/MacPython/h5py-wheels/blob/master/.travis.yml

**Comment by takluyver** [comment]
Yes, we're now building wheels for Intel Macs on Azure. We don't yet publish wheels for the new ARM Macs, though I imagine we will at some point (see #2008).

I've been aiming to build the wheels in our own CI as far as possible - it's nice to have it constantly tested so it just works at release time, without having to fiddle with things in a separate repo. But on the other hand, our CI is already pretty slow, with the product of platform, Python versions, HDF5 versions and numpy versions, and there's pressure to run even more flavours (like ARM Macs), so having some builds that we put off to release time might still be useful. :shrug:

**Comment by mattip** [comment]
NumPy is taking the lazy approach with a triggerable cibuildwheel task in the numpy repo CI. It only runs when a developer manually triggers the run (there is a Run Workflow button [here](https://github.com/numpy/numpy/actions/workflows/wheels.yml) if you have proper permissions) and nightly.

The code to do that is in a [separate workflow yml](https://github.com/numpy/numpy/blob/2f41a52d9b2fc70ea2bed6e04de470d854362ccd/.github/workflows/wheels.yml#L11)

```
on:
  schedule:
    # Nightly build at 1:42 UTC
    - cron: "42 1 * * *"
  push:
  pull_request:
  workflow_dispatch:

```

NumPy has wheels building now (including M1 arm64 via whatever magic cibuildwheel does). I am supposed to add the deploy part. As I commented on the [tracking issue](https://github.com/numpy/numpy/issues/20262#issuecomment-994400464) we will continue to use travis to build the aarch64 wheels since they cannot be easily tested via emulation.
